lib/quoters/WebhookQuoter.ts

```diff
@@ -60,7 +61,7 @@ export class WebhookQuoter implements Quoter {
     const quotes = await Promise.all(enabledEndpoints.map((e) => this.fetchQuote(e, request)));
```

Comment:
Should we use `allSettled` here as well?

Reply:
i dont think its necessary because fetchQuote can never reject, though good practice to